recipes/spyrmsd/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "spyrmsd" %}
+{% set version = 0.3.0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 985861283513fcf9979c028ccd4a9c8dfb7771673a77a8182104de618950e261
+
+build:
```

Comment:
If no C-extensions are being built and there are no platform specific dependencies, please consider making this `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

You will also want to change the `host` and `run` requirements for `python` to `python >=3.6`.

Reply:
So, the problem is that `graph_tool` is only available for macOS and Linux. I'll try to find a way to fallback to `NetworkX` for Windows.

Reply:
I went for `NetworkX` as default, so that everything can be installed in the same way.